src/components/date/date.js

```diff
@@ -447,27 +450,42 @@ class Date extends React.Component {
     return this.inputIconHTML('calendar');
   }
 
- /**
+  /**
+  * Returns the disabled array of days specified by props maxDate and minDate
+  *
+  * @method disabledDays
+  * @return {Array}
+  */
+  disabledDays() {
```

Comment:
I wouldn't worry about adding too much extra functionality to the datepickers if we don't already support it.

Did we already support this?